ui/components/Lifecycle/General/transfer-list/index.js

```diff
@@ -1,6 +1,5 @@
 import React, { useEffect, useState } from 'react';
```

Comment:
@amitamrutiya i think this transferlist can also moved to archive or completely remove because we are using this component from sistent and no longer from here.